runtime/wasm/src/module/mod.rs

```diff
@@ -1424,34 +1424,20 @@ impl<C: Blockchain> WasmInstanceContext<C> {
     /// function arweave.transactionData(txId: string): Bytes | null
     pub fn arweave_transaction_data(
         &mut self,
-        tx_id: AscPtr<AscString>,
+        _tx_id: AscPtr<AscString>,
     ) -> Result<AscPtr<Uint8Array>, HostExportError> {
-        if !self.experimental_features.allow_non_deterministic_arweave {
-            return Err(HostExportError::Deterministic(anyhow!(
-                "`arweave.transactionData` is deprecated. Improved support for arweave may be added in the future"
-            )));
-        }
-        let tx_id: String = asc_get(self, tx_id)?;
-        let data = self.ctx.host_exports.arweave_transaction_data(&tx_id);
-        data.map(|data| asc_new(self, &*data).map_err(Into::into))
-            .unwrap_or(Ok(AscPtr::null()))
+        Err(HostExportError::Deterministic(anyhow!(
```

Comment:
We don't need to check for the features anymore?

Reply:
No, the error is unconditional.